recipes/tracy/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class TracyConan(ConanFile):
+    name = "tracy"
+    description = "C++ frame profiler"
+    topics = ("tracy", "profiler", "performance", "gamedev")
+    homepage = "https://github.com/wolfpld/tracy"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = ["BSD 3-clause"]
+    settings = "os", "compiler", "build_type", "arch"
+
+    # Existing CMake tracy options with default value
+    tracy_options = {
+        "enable": ([True, False], True),
```

Comment:
Enable what? Need to improve this option name.

Reply:
This is the name used by the project, so I think it's better to stick with it as it will be prefixed by `tracy` in conan anyways.
cf https://github.com/wolfpld/tracy/blob/a0ae37a350aa825967166fc5ef4d64ea43b5b0ba/CMakeLists.txt#L36

Reply:
Perhaps? But yeah following upstream is never bad

```suggestion
        "enable_profiling": ([True, False], True),
```